smartcontractkit/external-adapters-js
# Fix Overrider casing normalization

## Description

Fixes a bug in Overrider for Coinpaprika and Coingecko where a symbol that had mixed casing was not matching.

## Changes
- Casing is normalized in the Overrider

## Steps to Test

1. Run the Coinpaprika adapter from `develop`
2. Request
```
{
    "data": {
         "from": "OHMv2",
        "overrides": {
          "coinpaprika": {
            "OHMv2": "ohmv2-olympus-v2"
          },
        },
        "to": "USD"
    }
}
```
3. The request should fail
4. Re-try from this branch, the request will succeed.

## Quality Assurance
